src/components/Fonts/FontPreLoader.tsx

```diff
@@ -2,17 +2,15 @@ import React from 'react';
 
 import Head from 'next/head';
 
+import { FONT_CDN } from 'src/utils/fontFaceHelper';
+
 const DEFAULT_LOCALE = 'en';
 
 const LOCALE_PRELOADED_FONTS = {
-  [DEFAULT_LOCALE]: [
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_regular.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_semibold.woff2' },
-    { type: 'font/woff2', location: 'ProximaVara/proxima_vara_bold.woff2' },
-  ],
-  ar: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
-  bn: [{ type: 'font/ttf', location: 'NotoSerifBengali/NotoSerifBengali-Regular.ttf' }],
-  ur: [{ type: 'font/woff2', location: 'urdu/NotoNaskhArabic-Regular.woff' }],
+  [DEFAULT_LOCALE]: [{ type: 'font/woff2', location: 'lang/ProximaVara/ProximaVara.woff2' }],
```

Comment:
We were using `proxima_vara_semibold` and `proxima_vara_bold` intentionally for non-default `font-weight` values.

Reply:
It seems the new font being used is less in size that the 3 files we used to use collectively:

Before:
<img width="1283" alt="Screen Shot 2022-01-25 at 09 39 32" src="https://user-images.githubusercontent.com/15169499/150900919-521adc05-e0c0-4ca9-8ce1-1a82c8e6b3a8.png">

After:
<img width="1283" alt="Screen Shot 2022-01-25 at 09 39 27" src="https://user-images.githubusercontent.com/15169499/150900943-23bf751a-3ba0-4594-b649-26f53ab3957f.png">